v3-docs/v3-migration-docs/breaking-changes/index.md

```diff
@@ -0,0 +1,207 @@
+  }
+});
+
+await Meteor.callAsync('getAllData') // [!code highlight]
+await Meteor.callAsync('otherMethod') // [!code highlight]
+
+```
+
+## Changes in Webapp
+
+::: tip TL;DR
+
+Webapp now uses Express under the hood. This means that you can use all the express features in your Meteor app.
+
+But if you did any customizations in the `WebApp` package, you should check if they are compatible with Express.
+
+:::
+
+
+The `webapp` package now exports this new properties:
+
+```ts
+type ExpressModule = {
+  (): express.Application;
+  json: typeof express.json;
+  raw: typeof express.raw;
+  Router: typeof express.Router;
+  static: typeof express.static;
+  text: typeof express.text;
+  urlencoded: typeof express.urlencoded;
+};
+
+export declare module WebApp {
+  // ...
+  /**
+   * @deprecated use handlers instead
+   */
+  var connectHandlers: express.Application;
+  var handlers: express.Application; // [!code highlight]
+  /**
+   * @deprecated use rawHandlers instead
+   */
+  var rawConnectHandlers: express.Application;
+  var rawHandlers: express.Application;
+  var httpServer: http.Server;
+  var expressApp: express.Application;
+  var express: ExpressModule; // [!code highlight]
+  // ...
+}
+
+// import { WebApp } from 'meteor/webapp';
+```
+
+If you want to use express in your app, you can do it like this:
+
+```js
+import { WebApp } from 'meteor/webapp';
+
+const app = WebApp.express(); // [!code highlight] you can use as a normal express app
+
+app.get('/hello', (req, res) => {
+  res.send('Hello World');
+});
+
+WebApp.handlers.use(express);
+
+```
+
+The code below is an example of how you can use the `handlers` property to create a route in your app:
+
+```js
+import { WebApp } from 'meteor/webapp';
+
+WebApp.handlers.get('/hello', (req, res) => {
+  res.send('Hello World');
+});
+
+```
+Changed engine from connect to express and changed api naming to match express. See below:
+  - `WebApp.connectHandlers.use(middleware)` is now `WebApp.handlers.use(middleware)`
+  - `WebApp.rawConnectHandlers.use(middleware)` is now `WebApp.rawHandlers.use(middleware)`
+  - `WebApp.connectApp` is now `WebApp.expressApp`
+
+
+A few methods from WebApp internals are now async:
+
+  - `WebAppInternals.reloadClientPrograms()`
+  - `WebAppInternals.pauseClient()`
+  - `WebAppInternals.generateClientProgram()`
+  - `WebAppInternals.generateBoilerplate()`
+  - `WebAppInternals.setInlineScriptsAllowed()`
+  - `WebAppInternals.enableSubresourceIntegrity()`
+  - `WebAppInternals.setBundledJsCssUrlRewriteHook()`
+  - `WebAppInternals.setBundledJsCssPrefix()`
+  - `WebAppInternals.getBoilerplate`
+
+## Meteor.userAsync
+
+You should use `Meteor.userAsync` instead of `Meteor.user` in your code, especially if you
+want isomorphism or want to get your user in the server.
+
+```js
+// Before
+const user = Meteor.user(); // [!code error]
+// After
+const user = await Meteor.userAsync(); // [!code highlight]
+
+```
+
+
+## Community migration guides
```

Comment:
I suggest moving the "Community migration guides" reference from the "Breaking Changes" section to the Overview section or creating a new section specifically for external links.

Creating a new section we could add blog posts and videos on migrating projects to Meteor 3.0.